drivers/input/touchscreen_upper.c

```diff
@@ -297,65 +291,49 @@ static int touch_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
   return ret;
 }
 
-static int touch_poll(FAR struct file *filep,
-                      struct pollfd *fds, bool setup)
+static int touch_poll(FAR struct file *filep, struct pollfd *fds, bool setup)
 {
-  FAR struct inode *inode = filep->f_inode;
-  FAR struct touch_upperhalf_s *upper = inode->i_private;
+  FAR struct touch_openpriv_s *openpriv = filep->f_priv;
   pollevent_t eventset = 0;
   int ret;
-  int i;
 
-  ret = nxsem_wait(&upper->exclsem);
+  ret = nxsem_wait(&openpriv->locksem);
   if (ret < 0)
     {
       return ret;
     }
 
   if (setup)
     {
-      for (i = 0; i < CONFIG_INPUT_TOUCHSCREEN_NPOLLWAITERS; i++)
```

Comment:
So can we eliminate `INPUT_TOUCHSCREEN_NPOLLWAITERS` if that becomes unused now?

Reply:
done